performance/tests/Alert.test.js

```diff
@@ -90,14 +90,14 @@ describe('Alert', () => {
 		let avgFCP = 0;
 		let avgLCP = 0;
 		for (let step = 0; step < stepNumber; step++) {
-			const page = await testMultiple.newPage();
+			const alertPage = await testMultiple.newPage();
```

Comment:
This was changed because `page` was declared as global above. The same is for all components.